### DISCUSSION TITLE
Is it possible to ignore files that starts with a lowercase?

### DISCUSSION THREAD

**Discussion by limitless-dev** [eslint/eslint#17372] [Q&A] [answered]
I need to ignore files in specific folder that starts with lowercase letter. The files were installed through UI library and I don't want to miss with them, so it would be great to just ignore them as they all starts with lowercase.

I tried the following but it ignores everything:
```
ignorePatterns: [
    'src/components/[a-z]*.tsx',
  ]
```

What I need to achieve:
- `src/components/button.tsx`: should be ignored
- `src/components/ActionButton.tsx`: should not be ignored

**Comment by Rec0iL99** [comment]
Hi @limitless-dev, you could maybe have a folder that contains the files you want ESLint to ignore when linting.

```js 
ignorePatterns: [
   'src/components/ignored/*.tsx',
]
```

**Comment by mdjermanovic** [comment, marked as answer]
> I tried the following but it ignores everything:
> 
> ```
> ignorePatterns: [
>     'src/components/[a-z]*.tsx',
>   ]
> ```

In the eslintrc config system, ignore patterns are case-insensitive, so it doesn't seem there's a good solution to achieve this with ignore patterns.

Globs are case-sensitive, so you could change your eslint command to match only files that start with an uppercase letter:

```
eslint "src/components/[A-Z]*.tsx"
```

In the [new config system](https://eslint.org/docs/latest/use/configure/configuration-files-new), ignore patterns are case-sensitive, so this would work:

```js
// eslint.config.js
module.exports = [
    {
        ignores: ['src/components/[a-z]*.tsx']
    },
    {
        files: ['**/*.tsx'],
        // ... configuration for .tsx files
    }
];
```

**Reply by limitless-dev** [reply]
That's awesome, thank you.
